src/Entities/QueryApi/DataPlatformIndexerStatuses.jsx

```diff
@@ -36,9 +73,12 @@ function handleResults(result) {
       }
       let data = result.body.data;
       if (data) {
-        const statuses = data.indexer_state;
+        const statuses = data[`${indexerAccountLink}_${indexer}_sys_metadata`];
         if (statuses.length > 0) {
-          setIndexerStatuses(statuses);
+          const status = statuses.find((s) => s.attribute === "STATUS")?.value;
+          const blockHeight = statuses.find((s) => s.attribute === "LAST_PROCESSED_BLOCK_HEIGHT")?.value;
```

Comment:
Should there be some fallback value for these? It's "possible" for these values to not be there, either because its provisioning, or if something went wrong in provisioning, or if the indexer hasn't run once yet.

Reply:
I think they'll just be missing in the UI if they are missing in the data.